ot2/octopus-full-ssh.py

```diff
@@ -0,0 +1,664 @@
+from opentrons import protocol_api
+
+
+metadata = {
+    'apiLevel': '2.12',
+    'protocolName': 'OCTOPUS OT-2 Protocol: SSH-compatible, All Stages, Offset Calibrated',
+    'author': 'Octant',
+    'description': 'v1.0.0'
+}
+
+
+def run(protocol: protocol_api.ProtocolContext):
+    """
+    OT-2 Protocol for running select steps of the Twist 96-Plex protocol 
+    (formally iGenomX RIPTIDE) adapted for OCTOPUS and sequencing on 
+    Illumina MiSeq with the MiSeq Reagent Micro Kit v2 (300-cycles).
+
+    After running the main thermocycler protocol in "A Reaction" and adding EDTA, 
+    the samples can thereafter be processed by the OT-2 for the rest of 
+    the Twist 96-Plex protocol up to final validation/quantification 
+    for sequencer loading.
+
+    This specific layout allows for running on the OT-2 directly through SSH, 
+    includes offset-calibration templates, and runs through all stages.
+
+    The following equipment/labware are assumed:
+    1x  8-Channel P300 (GEN2)
+    1x  8-Channel P20 (GEN2)
+    1x  Magnetic Module
+    4x  Opentrons 96 Filter Tip Rack 200 uL
+    1x  Opentrons 96 Filter Tip Rack 20 uL
+    3x  Bio-Rad 96 Well Plate 200 uL PCR
+    2x  Applied Biosystems MicroAmp 384 Well Plate 40 uL
+    1x  NEST 96 Deepwell Plate 2mL
+    """
+    ### specify the number of plates to process: integer between 1 and 8 inclusive
+    number_of_plates = 0
+    ### specify True if running protocol on OT-2 command line, otherwise False
+    ssh_mode = True
+
+    protocol.set_rail_lights(True)
+    Run = OCTOPUS(protocol, ssh_mode)
+    Run.Pool(number_of_plates)
+    Run.Post_Pool()
+    protocol.set_rail_lights(False)
```

Comment:
These scripts were written using the [Opetnrons Python API](https://docs.opentrons.com/v2/writing.html) which requires a function named exactly `run` to contain the protocol. Because of this, it might be redundant to have this further wrapped in a main block.